src/LibraryViewExtension/Handlers/NodeItemDataProvider.cs

```diff
@@ -25,18 +25,22 @@ class LoadedTypeItem
         public string keywords { get; set; }
     }
 
-    class LoadedTypeData
+    class LoadedTypeData<T>
```

Comment:
That is how template means? If it does not work, I doubt the reason why this language feature exists.